utils/deploy_to_dev_env.sh

```diff
@@ -4,55 +4,107 @@
 # This can be achieved by using aws-vault and an amin role for example
 # aws-vault exec di-ipv-dev -- ./deploy_to_dev_env.sh
 
-if ! which jq >/dev/null; then
-  echo "Please install jq to use this script"
-fi
+declare -a stack_status=("UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE")
+declare -a environments=("dev-danw" "dev-danp" "dev-danh" "dev-chrisw" "dev-amrits" "dev-kerrr" "dev-lait" "dev-ianj" "dev-derrenw" "dev-patrickb" "dev-vamh" "dev-wojciecho" "dev-aprilm")
 
-if ! docker info > /dev/null 2>&1 ; then
-  echo "Docker engine is not running, please start it."
-  exit 1
-fi
+export region=`aws configure get region`
```

Comment:
As per `shellcheck` guidance please use `$()` rather than legacy back ticks
>                  ^-- SC2006 (style): Use $(...) notation instead of legacy backticks `...`.

Can you run `shellcheck` on this and correct as necessary?